src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+
+    public Ball getRandomBall() {
+        Color color = new ColorSupplier().getRandomColor();
+        int number = new Random().nextInt(100);

```

Comment:
1. Don't create several instances of the same class if you can use only one instance for your purposes
2. 
![image](https://user-images.githubusercontent.com/20326858/230043259-99ce9ab0-6050-450e-ac42-46136d8b45d9.png)